jdaviz/components/plugin_dataset_select.vue

```diff
@@ -68,8 +68,7 @@
 module.exports = {
   props: ['items', 'selected', 'label', 'hint', 'rules', 'show_if_single_entry', 'multiselect',
           'api_hint', 'api_hints_enabled']
-  }
-};
+}
```

Comment:
Keeping the semi-colon won't hurt, right?

Reply:
I thought that was the whole cause of the problem?

Reply:
Ah, I see, I totally missed the extra bracket.

Reply:
semi-colon is optional, the extra bracket was the syntax error resulting in the template failing to load (really wish we had a better automated way to catch these!)

Reply:
It is optional until someone adds more stuff in the `<script></script>`, right? Anyways... prob ok for now.